internal/config/config.go

```diff
@@ -17,8 +25,15 @@ type Config struct {
 	ClusterID      string
 	PprofPort      int
 	LeaderElection LeaderElection
-	PodName        string
-	NodeName       string
+
+	MonitorMetadata string `mapstructure:"monitor_metadata"`
```

Comment:
```suggestion
	MonitorMetadataPath string `mapstructure:"monitor_metadata"`
```

As far I can see this is path to the file but name is a bit confusing.